src/main/java/redis/clients/jedis/commands/JedisCommands.java

```diff
@@ -361,6 +361,11 @@ default String setex(String key, int seconds, String value) {
 
   String echo(String string);
 
+  /**
+   * @deprecated This method will be removed from this interface. Use
+   * {@link AdvancedJedisCommands#move(java.lang.String, int)}.
+   */
+  @Deprecated
   Long move(String key, int dbIndex);
```

Comment:
Won't it mark the method as deprecated on Jedis too?

Reply:
@gkorland No. The method will stay. It's just moved to Advanced... interface.